dspace-server-webapp/src/test/java/org/dspace/app/rest/builder/CollectionBuilder.java

```diff
@@ -155,14 +155,38 @@ public Collection build() {
 
     @Override
     public void cleanup() throws Exception {
+        context.turnOffAuthorisationSystem();
```

Comment:
This line should no longer be needed. Context auth is managed in the methods below it.

Reply:
This should be fixed now.